src/WorldIDIdentityManagerImplV1.sol

```diff
@@ -343,6 +359,32 @@ contract WorldIDIdentityManagerImplV1 is OwnableUpgradeable, UUPSUpgradeable, IW
         return _latestRoot;
     }
 
+    function sendRootToStateBridge() public virtual onlyProxy onlyInitialized {
+        if(stateBridgeProxy == address(0x0)) revert stateBridgeHasBeenTurnedOff();        
+
+        (bool success,) = stateBridgeProxy.call(
+            abi.encodeWithSignature("sendRootMultichain(uint256)", _latestRoot)
+        );
+
+        // If the call to the state bridge proxy failed, we revert with a failure.
+        if (!success) revert StateBridgeProxySendRootMultichainFailure();
+    }
+
+    /// @notice Allows a caller to query the address of the current stateBridgeProxy.
+    ///
+    /// @return proxy The address of the currently used stateBridgeProxy
+        function getStateBridgeProxy() public view virtual onlyProxy onlyInitialized returns (address proxy) {
```

Comment:
Call this `stateBridgeProxyAddress()` so it lines up with `latestRoot()`.

Reply:
I didn't name it like latestRoot before because there'd be a name clash with stateBridgeProxy. I guess I can name it stateBridgeProxyAddress and return stateBridgeProxy.